resources/translations/en.edn

```diff
@@ -217,43 +217,43 @@
+          :application-rejected {:subject-to-applicant "Your application %3 has been rejected"
+                                 :message-to-applicant "Dear %1,\n\nYour application %3 has been rejected.\n\nYou can view the application and the decision at %6"
+                                 :subject-to-handler "(%3) Application has been rejected"
+                                 :message-to-handler "Dear %1,\n\n%2 has rejected the application %3 from %4.\n\nYou can view the application and the decision at %6"}
+          :application-closed {:subject-to-applicant "Your application %3 has been closed"
+                               :message-to-applicant "Dear %1,\n\nYour application %3 has been closed.\n\nYou can view the application at %6"
+                               :subject-to-handler "(%3) Application has been closed"
+                               :message-to-handler "Dear %1,\n\n%2 has closed the application %3 from %4.\n\nYou can view the application at %6"}
+          :application-returned {:subject-to-applicant "Your application %3 needs to be amended"
+                                 :message-to-applicant "Dear %1,\n\nYour application %3 has been returned for your consideration. Please, amend according to requests and resubmit.\n\nYou can review the application at %6"
+                                 :subject-to-handler "(%3) Application has been returned to applicant"
+                                 :message-to-handler "Dear %1,\n\n%2 has returned the application %3 to the applicant %4 for modifications.\n\nYou can view the application at %6"}
+          :application-licenses-added {:subject "Your application's %3 terms of use have changed"
+                                       :message "Dear %1,\n\n%2 has requested your approval for changed terms of use to application %3.\n\nYou can review the application and approve the changed the terms of use at %6"}
+          :application-resubmitted {:subject "(%3) Application has been resubmitted"
+                                    :message "Dear %1,\n\nApplication %3 has been resubmitted by %4.\n\nYou can review the application at %6"}
+          :application-submitted {:subject "(%3) A new application has been submitted"
+                                  :message "Dear %1,\n\n%4 has submitted a new application %3 to access resource(s) %5.\n\nYou can review the application at %6"}
+          :comment-requested {:subject "(%3) Review request"
+                              :message "Dear %1,\n\n%2 has requested your review on application %3, submitted by %4.\n\nYou can review the application at %6"}
+          :commented {:subject "(%3) Application has been reviewed"
+                      :message "Dear %1,\n\n%2 has reviewed the application %3, submitted by %4.\n\nYou can review the application at %6"}
+          :decided {:subject "(%3) Application decision notification"
```

Comment:
This sounds very artificial and different from the others. How about "Decision has been filed/made/given"

Reply:
It's verbatim from the spec :)

Reply:
That's the problem with specs ;)